yarn.lock

```diff
@@ -8434,7 +8434,16 @@ postcss-will-change@^4.0.1:
   resolved "https://registry.yarnpkg.com/postcss-will-change/-/postcss-will-change-4.0.1.tgz#7604320aa8ba147e4d00732a37d32658aeb436d4"
   integrity sha512-nJIHJ+mwWviIPVs/vuUfa/w8XAOXUjfYSIfNgdnZdo9uHC96x4nrY2ByEkdwYLvnxKto09JYDSWH8sWYLoUhUA==
 
-postcss@^7.0.14, postcss@^7.0.17, postcss@^7.0.2, postcss@^7.0.21, postcss@^7.0.26, postcss@^7.0.31, postcss@^7.0.32, postcss@^7.0.35, postcss@^7.0.6:
+postcss@^7.0.14, postcss@^7.0.17, postcss@^7.0.2, postcss@^7.0.21, postcss@^7.0.26, postcss@^7.0.31, postcss@^7.0.32, postcss@^7.0.6:
+  version "7.0.35"
+  resolved "https://registry.yarnpkg.com/postcss/-/postcss-7.0.35.tgz#d2be00b998f7f211d8a276974079f2e92b970e24"
+  integrity sha512-3QT8bBJeX/S5zKTTjTCIjRF3If4avAT6kqxcASlTWEtAFCb9NH0OUxNDfgZSWdP5fJnBYCMEWkIFfWeugjzYMg==
+  dependencies:
+    chalk "^2.4.2"
+    source-map "^0.6.1"
+    supports-color "^6.1.0"
+
+postcss@^7.0.35:
```

Comment:
We need to dedupe this please :)

Reply:
This is after running `yarn-deduplicate`. And:

```
yarn why postcss
yarn why v1.22.15
[1/4] 🤔  Why do we have the module "postcss"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "postcss@7.0.35"
info Has been hoisted to "postcss"
info Reasons this module exists
   - "workspace-aggregator-5d44ef85-ba38-4644-99d2-9926cbbe65e0" depends on it
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint-order#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#postcss-less#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint-order#postcss-sorting#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#sugarss#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#postcss-sass#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#postcss-safe-parser#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#autoprefixer#postcss"
   - Hoisted from "_project_#@shopify#stylelint-plugin#stylelint#postcss-scss#postcss"
info Disk size without dependencies: "1.26MB"
info Disk size with unique dependencies: "2.17MB"
info Disk size with transitive dependencies: "2.37MB"
info Number of shared dependencies: 7
=> Found "@shopify/postcss-plugin#postcss@8.2.10"
info This module exists because "_project_#@shopify#postcss-plugin" depends on it.
info Disk size without dependencies: "312KB"
info Disk size with unique dependencies: "1.26MB"
info Disk size with transitive dependencies: "1.26MB"
info Number of shared dependencies: 3
```

Is it making a reappearance due to hoisting? Do I need to add a resolution? HALP.

Reply:
There's also already a `postcss@^8.2.8` in there.